app/controllers/calendar_controller.rb

```diff
@@ -0,0 +1,28 @@
+class CalendarController < ApplicationController
+  skip_before_action :authenticate_user!
+
+  def show
+    @title = "Calendar"
+    @events = Event.all
+    render 'show.html.erb'
+  end
+
+  def new_event
+    render 'new'
+  end
+
+  def create
+    date = "#{params[:date]['month']}/#{params[:date]['day']}/#{params[:date]['year']}"
+    strp_format = "%m/%d/%Y"
+    start_time = DateTime.strptime(date, strp_format)
```

Comment:
Shouldn't need to convert to a date first - use the rails `select_datetime` in the form and just use the param that is passed in from that